met-api/src/met_api/services/user_service.py

```diff
@@ -24,13 +25,16 @@ def create_or_update_user(self, user: UserSchema):
         external_id = user.get('external_id')
         db_user = UserModel.get_user_by_external_id(external_id)
         if db_user is None:
+            is_staff_user = user.get('email_id', '').endswith('gov.bc.ca')
```

Comment:
I don't think this will do, lets say I'm from ministry of health and used my gov email to answer a survey, this will think i'm a staff user, the way we are handling this for now is using the external_id which is the idir Id in this case, if null then its public. We might have to have a flag for this in the future